test/e2e/ingress/secret.go

```diff
@@ -144,6 +144,6 @@ RU+QPRECgYB6XW24EI5+w3STbpnc6VoTS+sy9I9abTJPYo9LpCJwfMYc9Tg9Cx2K
 		tlsUpdate, err := s.ListApisixTls()
 		assert.Nil(ginkgo.GinkgoT(), err, "list tlsUpdate error")
 		assert.Len(ginkgo.GinkgoT(), tlsUpdate, 1, "tls number not expect")
-		assert.NotEqual(ginkgo.GinkgoT(), KeyInApisix, tlsUpdate[0].Key, "tls key not expect")
+		assert.Equal(ginkgo.GinkgoT(), certUpdate, tlsUpdate[0].Cert, "tls cert not expect")
```

Comment:
We may also add some test cases to cover the ApisixTls delete.

Reply:
Done